src/sentry/integrations/source_code_management/metrics.py

```diff
@@ -92,3 +108,22 @@ def get_extras(self) -> Mapping[str, Any]:
             "organization_id": (self.organization.id if self.organization else None),
             "org_integration_id": (self.org_integration.id if self.org_integration else None),
         }
+
+
+@dataclass
+class SourceCodeSearchEndpointInteractionEvent(IntegrationEventLifecycleMetric):
```

Comment:
like above, i think we can use the common SCMInteractionEvent after we fix the merge conflict